src/components/Pf/PfTitle.tsx

```diff
@@ -90,18 +90,16 @@ class PfTitle extends React.Component<PfTitleProps, PfTitleState> {
     return (
       <h2 className={PfTitleStyle}>
         {this.state.icon} {this.state.name}
-        {this.props.istio ? (
-          !this.props.hideGraphLink && (
-            <>
```

Comment:
If we are never going to use  this.props.hideGraphLink we should remove from https://github.com/kiali/kiali-ui/blob/2b5aef79176372e58336e8d95343e7ffa18677e5/src/components/Pf/PfTitle.tsx#L22

Reply:
If we are going to show the link only if there is a name it's ok for me. But I think that we don't need that check  we always expect a name in the url

Reply:
+1 I don't even know where that came from. I'll remove.

Reply:
It was in the code I didn't know if the author of the graph link wanted to control this